bin/install

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+set -eu
+
+# Set ANSI Codes
+Off='\033[00m'
+Bold='\033[1m'
+
+Red='\033[0;31m'
+Blue='\033[0;36m'
+Green='\033[0;32m'
+Purple='\033[0;35m'
+Blue='\033[0;36m'
+
+abort() {
+  printf "%s\n" "$@" >&2
+  exit 1
+}
+
+# Fail fast if tileserver configuration isn't found
```

Comment:
Just to be clear, I would say "if map resources are not found in the tileserver directory". Because "tileserver configuration" generally refers to the `config.json` file (which is unique to tileserver), which we are still providing by default.